src/docs/src/components/Prism.svelte

```diff
@@ -7,22 +7,24 @@
 
 <script>
   import "prism-svelte"
-  import { tick } from "svelte"
+  import { afterUpdate } from "svelte"
 
   export let language = "javascript"
   export let source = ""
   export let transform = (x) => x
   let element, formattedCode
 
-  $: $$props && (source || element) && highlightCode()
```

Comment:
replace by the `afterUpdate` below. This is to support correct re-rendering when the dynamic global prefix value changes (element won't change here, so the content is not updated accordingly)